ui/src/app/dashboard/dashboard.component.spec.ts

```diff
@@ -187,8 +197,14 @@ describe('DashboardComponent', () => {
   it('should create expected amount of cards', fakeAsync(() => {
     const summaryCardNum = de.queryAll(By.css('jm-total-summary mat-card')).length;
     const groupedCardNum = de.queryAll(By.css('jm-grouped-summary')).length;
+    const statusArray: Array<JobStatus> = [JobStatus.Succeeded, JobStatus.Aborted, JobStatus.Running, JobStatus.Failed];
 
-    expect(summaryCardNum).toEqual(TEST_AGGREGATION_RESPONSE.summary.counts.length);
+    for (let countEntry of TEST_AGGREGATION_RESPONSE.summary.counts) {
```

Comment:
I'd prefer just dropping this new test code if you can't find a better way of testing this. Currently, this is pretty much a copy-paste of the real component code, so it doesn't provide much test value except as a code change detector.

A hardcoded number with a comment explaining would be an improvement, or just dropping it.

Reply:
Actually I do not need to paste the real code. Hard-coded 4 still making sense here.